python_ta/transforms/setendings.py

```diff
@@ -225,7 +264,16 @@ def set_arguments(node):
     """
     if _get_last_child(node):
         set_from_last_child(node)
-    else:
+    else:  # node does not have children.
+        # set from col offset of parent node, plus len of name, etc.
+        # Note: if there are no arguments, this node takes up no space
+        parent_node = node.parent
+        if isinstance(parent_node, astroid.FunctionDef):
```

Comment:
These changes still suffer from whitespace issues. This means you should revert these changes, and the problems should be handled by examining the source code.